thirdparty/kpvcrlib/CMakeLists.txt

```diff
@@ -58,6 +58,11 @@ endif()
 
 if(DEFINED ENV{LEGACY})
     add_definitions(-DDISABLE_CLOEXEC)
+else()
+    # Enable LFS Support in glibc
+    add_definitions(
+        -D_LARGEFILE64_SOURCE=1
+    )
```

Comment:
So, no `DISABLE_CLOEXEC` on non-legacy ? :)

I would also explicitely name this -D `HAS_STAT64` here and in cre if we don't need it for more things on the cre side - and to avoid any conflict with other bits of code (or even a `-D_CRE_CAN_USE_STAT64` to avoid more conflicts if HAS_STAT64 is too plain)

Reply:
Yeah, we *want* (and know that it's supported) CLOEXEC on other platforms ;).